packages/office-ui-fabric-react/src/components/TextField/TextField.base.tsx

```diff
@@ -148,7 +148,10 @@ export class TextFieldBase extends BaseComponent<ITextFieldProps, ITextFieldStat
         }
       );
 
-      this._delayedValidate(newProps.value);
+      const { validateOnFocusIn, validateOnFocusOut } = newProps;
+      if (!(validateOnFocusIn || validateOnFocusOut)) {
+        this._delayedValidate(newProps.value);
```

Comment:
You're tweaking some logic here, it would be great to clarify in the documentation of the props.

Reply:
And perhaps have some validation examples that illustrate a concept! Otherwise devs just need to guess what's going on.

Reply:
Actually I think this is just making the whole component consistent.. this is what I would expect based on the `validateOnFocus` props that are made available. All other calls to `_validate` and `_delayValidate` respect these props, this is the one case that didn't.